project_national/panel_upgrades.yml

```diff
@@ -0,0 +1,1366 @@
+    build_existing_model:
+      simulation_control_timestep: 15
+      simulation_control_run_period_begin_month: 1
+      simulation_control_run_period_begin_day_of_month: 1
+      simulation_control_run_period_end_month: 12
+      simulation_control_run_period_end_day_of_month: 31
+      simulation_control_run_period_calendar_year: 2007
+
+    emissions:
+      - scenario_name: LRMER_MidCase_15
+        type: CO2e
+        elec_folder: data/cambium/2022/LRMER_MidCase_15
+
+    utility_bills:
+      - scenario_name: Bills
+
+    simulation_output_report:
+      timeseries_frequency: timestep
+      include_timeseries_total_consumptions: true
+      include_timeseries_fuel_consumptions: true
+      include_timeseries_end_use_consumptions: true
+      include_timeseries_emissions: true
+      include_timeseries_emission_fuels: true
+      include_timeseries_emission_end_uses: true
+      include_timeseries_hot_water_uses: true
+      include_timeseries_total_loads: true
+      include_timeseries_component_loads: true
+      include_timeseries_unmet_hours: true
+      include_timeseries_zone_temperatures: true
+      include_timeseries_airflows: true
+      include_timeseries_weather: true
+      include_timeseries_resilience: true
+
+    reporting_measures:
+      - measure_dir_name: QOIReport
+
+    server_directory_cleanup:
+      retain_in_idf: false
+      retain_schedules_csv: false
+
+baseline:
+  n_buildings_represented: 140382740 # American Community Survey 2022 5-year, B25001, contiguous US + AK
+
+references:
+
+    options:
+    #### 1.0 Common Options For Heat Pump, 12 options ####
+      - &1_heating_fuel
+        option: Heating Fuel|Electricity
+        apply_logic:
+          - not: Heating Fuel|None
+
+      - &2_cooling_efficiency_ducted
+        option: HVAC Cooling Efficiency|Ducted Heat Pump
+        apply_logic: &logic_ducted_hvac_no_ashp
+          - HVAC Has Ducts|Yes
+          - not: Heating Fuel|None
+          - not: HVAC Heating Type And Fuel|Electricity ASHP
+      
+      - &3_cooling_efficiency_ductless
+        option: HVAC Cooling Efficiency|Non-Ducted Heat Pump
+        apply_logic: &logic_ductless_hvac_no_mshp
+          - HVAC Has Ducts|No
+          - not: Heating Fuel|None
+          - not: HVAC Heating Type And Fuel|Electricity MSHP
+
+      - &4_100_percent_conditioned
+        option: HVAC Cooling Partial Space Conditioning|100% Conditioned
+        apply_logic:
+          - or:
+            - *logic_ducted_hvac_no_ashp
+            - *logic_ductless_hvac_no_mshp
+      
+      - &5_cooling_setpoint_no_offset
+        option: Cooling Setpoint Has Offset|No
+        apply_logic:
+          - or:
+            - *logic_ducted_hvac_no_ashp
+            - *logic_ductless_hvac_no_mshp
+
+      - &6_cooling_setpoint_offset_magnitude_0
+        option: Cooling Setpoint Offset Magnitude|0F
+        apply_logic:
+          - or:
+            - *logic_ducted_hvac_no_ashp
+            - *logic_ductless_hvac_no_mshp
```

Comment:
Add `- not: Cooling Setpoint Offset Magnitude|0F` similar to above.
Again the logic is not to touch buildings that are already that way for accurate applicable count.